Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -376,6 +369,62 @@ CGImageRef CGImageCreateWithMaskingColors(CGImageRef image, const CGFloat* compo
 
 #pragma region WIC_HELPERS
 
+IWICBitmap* _CGImageGetImageSource(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    return image->ImageSource().Get();
+}
+
+DisplayTexture* _CGImageGetDisplayTexture(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+
+    ComPtr<ICGDisplayTexture> displayTextureAccess;
+    RETURN_NULL_IF_FAILED(image->ImageSource().Get()->QueryInterface(IID_PPV_ARGS(&displayTextureAccess)));
+    RETURN_NULL_IF(!displayTextureAccess);
+
+    IDisplayTexture* displayTexture;
+    RETURN_NULL_IF_FAILED(displayTextureAccess->DisplayTexture(&displayTexture));
+    RETURN_NULL_IF(!displayTexture);
+
+    return displayTexture->GetTexture();
+}
+
+// Return the data pointer to the Image data.
+void* _CGImageGetRawBytes(CGImageRef image) {
+    RETURN_NULL_IF(!image);
```

Comment:
do you expect to be passed null in any of these methods?  if not, we should remove the null checks and get a much easier to debug AV.